--save is not required anymore since version 5 (#1661)
nodejs/nodejs.dev: src/documentation/0016-npm/index.md

npm install <package-name>
```

> Furthermore, since npm 5, this command add `<package-name>` to the `package.json` file _dependencies_. Before version 5, you needed to add the flag `--save`.

Reviewer: ```suggestion
Furthermore, since npm 5, this command adds `<package-name>` to the `package.json` file _dependencies_. Before version 5, you needed to add the flag `--save`.
```

Often you'll see more flags added to this command:

* `--save-dev` installs and adds the entry to the `package.json` file _devDependencies_

> * `--no-save` installs but do not adds the entry to the `package.json` file _dependencies_

Reviewer: ```suggestion
* `--no-save` installs but does not add the entry to the `package.json` file _dependencies_
```
